app/views/layouts/_user_menu.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="col-md-3 mu-tab-body mu-user-menu">
+  <div>
+    <div class="mu-user-menu-section">
+      <div class="mu-user-menu-header">
+        <%= t(:my_account) %>
+      </div>
+      <div class="mu-user-menu-item">
+        <%= profile_user_menu_link %>
+      </div>
+    </div>
+    <div class="mu-user-menu-section">
+      <div class="mu-user-menu-header">
+        <%= t(:messages) %>
+      </div>
+      <div class="mu-user-menu-item">
+        <%= messages_user_menu_link %>
+      </div>
+      <% if should_show_discussions? %>
```

Comment:
It's up to you whether to extract it to a helper or not but we should use `user&.can_discuss_here?` which covers more cases other than if the organization has the forum enabled

Reply:
I thought of extracting it but it really made no sense to do

```ruby
def can_discuss_here?(user)
  user&.can_discuss_here?
end
```

In fact, I don't think the `&` is even needed in this case as you can't access that view without being logged in.

Reply:
Yep, I totally agree. I made that comment when there was a helper called `should_show_discussions?` that tested only `forum_enabled?` for current orga.